samples/consuming-tas-utility/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,37 @@
+# Set root logger level to error
+#log4j.rootLogger=${app.log.root.level}, Console, File
+rootLogger.level=INFO
+rootLogger.appenderRef.stdout.ref=ConsoleAppender
+rootLogger.appenderRef.rolling.ref=RollingAppender
+
+###### Console appender definition #######
+# All outputs currently set to be a ConsoleAppender.
+# Direct log messages to stdout
+appender.ConsoleAppender.type=Console
+appender.ConsoleAppender.name=ConsoleAppender
+appender.ConsoleAppender.layout.type=PatternLayout
+# use log4j NDC to replace %x with tenant domain / username
+appender.ConsoleAppender.layout.pattern=%d{ISO8601} %x %-5p [%c{3}] [%t] %replace{%m}{[\r\n]+}{}%n
+#log4j.appender.Console.layout.ConversionPattern=%d{ABSOLUTE} %-5p [%c] %m%n
+
+###### File appender definition #######
+# Direct log messages to a log file
+appender.RollingAppender.type=RollingFile
+appender.RollingAppender.name=RollingAppender
+appender.RollingAppender.fileName=./target/reports/alfresco-tas.log
+appender.RollingAppender.filePattern=./target/reports/alfresco-tas.log.%d{yyyy-MM-dd}
+appender.RollingAppender.layout.type=PatternLayout
+appender.RollingAppender.layout.pattern=%d{yyyy-MM-dd} %d{ABSOLUTE} %-5p [%c] [%t] %replace{%m}{[\r\n]+}{}%n
+appender.RollingAppender.policies.type = Policies
+appender.RollingAppender.policies.time.type = TimeBasedTriggeringPolicy
+appender.RollingAppender.policies.time.interval = 1
```

Comment:
In the original file the rolling strategy was not based on time, but rather on size combined with a max backup count similar to this:

```
###### File appender definition #######
appender.rolling.type=RollingFile
appender.rolling.name=RollingAppender
appender.rolling.fileName=./target/reports/alfresco-tas.log
appender.rolling.filePattern=./target/reports/alfresco-tas.log.%i
appender.rolling.layout.type=PatternLayout
appender.rolling.layout.pattern=[%t] %d{HH:mm:ss} %-5p %c{1}:%L - %replace{%m}{[\r\n]+}{}%n
appender.rolling.policies.type=Policies
appender.rolling.policies.size.type=SizeBasedTriggeringPolicy
appender.rolling.policies.size.size=10MB
appender.rolling.strategy.type=DefaultRolloverStrategy
appender.rolling.strategy.max=10
```